### ISSUE TITLE
Curating a list of awesome Communities

### ISSUE THREAD

**Issue by ashwinexe** [EddieHubCommunity/support#1811]
**Description**

There are many communities like **EddieHub** which have amazing developers and members waiting for us to join and participate with them. As an Idea, Its good to have a curated list of **Communities worth exploring** at one place so anyone can take a look and join them, saving a lot of time spent in research for this on the internet.

** Additional Thoughts**
- Creating a discussion on communities can **attract possible spam**.
- Would a readme on the repo could be effective?
- Need to come up with an efficient way to avoid any of such issues
- Need to verify the communities before adding them to the list
- Is it a good idea to provide **links** to join them directly or links to their web page, if present ?
- Should we categorize the communities? (Open source, Topic focused, hackathon based etc. )

But before that join our community [here](http://discord.eddiehub.org) :)

**Comment by Akshu-on-github** [comment]
My two cents:
- FreeCodeCamp
- Nick's Corner
- Devs'Street (I'm biased towards this one though!)

**Comment by Vyvy-vi** [comment]
This could be great!, however, I'm concerned if this could potentially lead to people spamming their own content, without actually building or supporting the community.

And the verification process might be a bit hefty. Perhaps, we could compile such a list from community collaborations?

**Comment by ashwinexe** [comment]
> This could be great!, however, I'm concerned if this could potentially lead to people spamming their own content, without actually building or supporting the community.
> 
> And the verification process might be a bit hefty. Perhaps, we could compile such a list from community collaborations?

Yes, I also pointed out the possibility of spam in my first point. This discussion is aiming at **how should be approach** to make a list of communities and not meant to have the actual list. Perhaps I should be more clear in description.

**Comment by krishdevdb** [comment]
> > This could be great!, however, I'm concerned if this could potentially lead to people spamming their own content, without actually building or supporting the community.
> > And the verification process might be a bit hefty. Perhaps, we could compile such a list from community collaborations?
> 
> Yes, I also pointed out the possibility of spam in my first point. This discussion is aiming at **how should be approach** to make a list of communities and not meant to have the actual list. Perhaps I should be more clear in description.

We should have a list only changeable by trusted and high-value people! That list can be in a repository or seated in our current repo, as the discord-readme.md is hanging. I recommend it to include partners, more communities like freecodecamp, nhcommunity... we can also add a regional-based page/file for local communities like SurPathHub by @vinzvinci. That might be enough! We can also only people with roles > high value, to be able to open issues for some more awesome communities that can be added!

**Comment by ashwinexe** [comment]
> > > This could be great!, however, I'm concerned if this could potentially lead to people spamming their own content, without actually building or supporting the community.
> > > And the verification process might be a bit hefty. Perhaps, we could compile such a list from community collaborations?
> > 
> > Yes, I also pointed out the possibility of spam in my first point. This discussion is aiming at **how should be approach** to make a list of communities and not meant to have the actual list. Perhaps I should be more clear in description.
> 
> We should have a list only changeable by trusted and high-value people! That list can be in a repository or seated in our current repo, as the discord-readme.md is hanging. I recommend it to include partners, more communities like freecodecamp, nhcommunity... we can also add a regional-based page/file for local communities like SurPathHub by @vinzvinci. That might be enough! We can also only people with roles > high value, to be able to open issues for some more awesome communities that can be added!

That's a good idea, now how should we approach the community and discuss this with everyone? (Which includes all high value and above people). Is it a good topic for community Calls or should we look for another way.

**Comment by krishdevdb** [comment]
> > > > This could be great!, however, I'm concerned if this could potentially lead to people spamming their own content, without actually building or supporting the community.
> > > > And the verification process might be a bit hefty. Perhaps, we could compile such a list from community collaborations?
> > > 
> > > 
> > > Yes, I also pointed out the possibility of spam in my first point. This discussion is aiming at **how should be approach** to make a list of communities and not meant to have the actual list. Perhaps I should be more clear in description.
> > 
> > 
> > We should have a list only changeable by trusted and high-value people! That list can be in a repository or seated in our current repo, as the discord-readme.md is hanging. I recommend it to include partners, more communities like freecodecamp, nhcommunity... we can also add a regional-based page/file for local communities like SurPathHub by @vinzvinci. That might be enough! We can also only people with roles > high value, to be able to open issues for some more awesome communities that can be added!
> 
> That's a good idea, now how should we approach the community and discuss this with everyone? (Which includes all high value and above people). Is it a good topic for community Calls or should we look for another way.

As per me, the community call is not enough! We need a good decision before we do that cause spam is a thing really to discuss upon! Moreover, the talks in the call will not be stored anywhere but if we keep all our talk on a single issue like this one! Everything will be documented thus the decision process will be more analytical and secure!

**Comment by vinzvinci** [comment]
Community call **timezone** is not friendly to some Asians. It is midnight or 1am here.

**Comment by krishdevdb** [comment]
> Community call **timezone** is not friendly to some Asians. It is midnight or 1 am here.

Yep! Another Point. In India, it is 10:30 pm but if we go more east than India, it will be 😵

**Comment by Vyvy-vi** [comment]
hmm, I would like to hear from Eddie about this 🤔

**Comment by Sam-Varghese** [comment]
## My Idea

How about listing only those communities that have at least 100 members

## Reason

Having at least these many members indicates that the community is popular and that there is something special in that community that is attracting more people.

## Advantages

- No spams as members would be listing only those communities that have earned a good name.
- Responsibility for updating this list would not be on the shoulders of high-value community members only.
- This would ensure that some active members who are aware of some great newly formed communities also get an opportunity to share about that with all members of EddieHub.

**Comment by Vyvy-vi** [comment]
> ## My Idea
> How about listing only those communities that have at least 100 members
> 
> ## Reason
> * Having at least these many members indicates that the community is popular and that there is something special in that community that has attracted these many members.
> * More members in the community mean more activity, discussions, and development going on.
> 
> ## Advantages
> * No spams as members would be listing only those communities that have earned a good name.
> * Responsibility for updating this list would not be on the shoulders of high-value community members only.
> * This would ensure that active members who are aware of some great newly formed communities also get an opportunity to share about that with all members of EddieHub.
> * This would engage the whole EddieHub community and not just high-value community members.
> 
> What are your opinions about this idea?

that sounds like a good idea :+1:
Perhaps we could have a thing where we list communities, after interacting with them on youtube/calls(or if they have some sort of reputation), and keep the repo open for contributions(anyone can open a PR about their community, and if it's good, we could add them by merging the Pr, or we could link this with Eddie's contact form)

**Comment by ashwinexe** [comment]
> What about adding [The Algorithms](https://the-algorithms.com)? I think it is a good community. Let me know what do you think, though. 🙂

that's a sweet (and very handy) website right there! I never knew about it. Thanks for sharing

**Comment by krishguptadev** [comment]
Also please include the discord invites for these communities it took me a lot of time to find the SidHacks server not sure why they don't display it on a big banner 🥲
